### PR TITLE
New package pass-update

### PR DESCRIPTION
**Pull request by adbrown101**
I would like to include the pass-update extension to the void packages

### CODE REVIEW

File: srcpkgs/pass-update/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'pass-update'
+pkgname=pass-update
+version=2.1
+revision=1
+archs=noarch
+build_style=gnu-makefile
+depends="pass"
+short_desc="A pass extension providing an easy flow for updating passwords"
```
Comment (Hoshpak) [inline srcpkgs/pass-update/template]: Please remove the article, according to our convention descriptions should not start with one. Just `Pass extension providing an easy flow for updating passwords` is fine.

File: srcpkgs/pass-update/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+maintainer="Alan Brown <adbrown@rocketmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/roddhjav/pass-update"
+distfiles="https://github.com/roddhjav/pass-update/releases/download/v${version}/pass-update-${version}.tar.gz"
+checksum=2b0022102238e022e7ee945b7622f4c270810cda46508084fcb193582aafaf6f
+
+post_install() {
+	vmkdir usr/share/bash-completion/completions
+	mv ${DESTDIR}/etc/bash_completion.d/* ${DESTDIR}/usr/share/bash-completion/completions
+}
```
Comment (zdtcd) [inline srcpkgs/pass-update/template]: This block of code could be replaced by this line:
```
make_install_args=BASHCOMPDIR=/usr/share/bash-completion/completions
```

File: srcpkgs/pass-update/template (hunk 1 above)
Comment (adbrown101) [reply srcpkgs/pass-update/template]: Changed this

File: srcpkgs/pass-update/template (hunk 2 above)
Comment (adbrown101) [reply srcpkgs/pass-update/template]: Changed this.

**Pull request merged**